ckanext/xloader/jobs.py

```diff
@@ -413,36 +404,20 @@ def set_datastore_active(data, resource, logger):
     set_resource_metadata(update_dict=data)
 
 
-def callback_xloader_hook(result_url, api_key, job_dict):
-    '''Tells CKAN about the result of the xloader (i.e. calls the callback
+def callback_xloader_hook(job_dict):
+    '''Tells CKAN about the result of the xloader (i.e. calls the action
     function 'xloader_hook'). Usually called by the xloader queue job.
-    Returns whether it managed to call the sh
-    '''
-    api_key_from_job = job_dict.pop('api_key', None)
-    if not api_key:
-        api_key = api_key_from_job
-    headers = {'Content-Type': 'application/json'}
-    if api_key:
-        if ':' in api_key:
-            header, key = api_key.split(':')
-        else:
-            header, key = 'Authorization', api_key
-        headers[header] = key
-
-    log = logging.getLogger(__name__)
-    log.info("Sending data to xloader hook callback at: %s", result_url)
 
+    Returns whether it managed to call the xloader_hook action or not
+    '''
     try:
-        result = requests.post(
-            result_url,
-            data=json.dumps(job_dict, cls=DatetimeJsonEncoder),
-            verify=SSL_VERIFY,
-            headers=headers)
-    except requests.ConnectionError:
-        log.warning("Failed to call xloader hook callback at: %s", result_url)
+        get_action('xloader_hook')(get_xloader_user_context(), job_dict)
+    except Exception as e:
+        log = logging.getLogger(__name__)
+        log.warning("Failed to call xloader_hook action: %s", e)
```

Comment:
would be useful to emit the traceback to the logs too, especially since we're swallowing it here

Reply:
passing logger into this method now so it should go to the log file and other log handlers setup